instrumentation/apache-camel-2.20/javaagent/src/test/groovy/io/opentelemetry/javaagent/instrumentation/apachecamel/SingleServiceCamelTest.groovy

```diff
@@ -11,7 +11,7 @@ import io.opentelemetry.instrumentation.test.AgentInstrumentationSpecification
 import io.opentelemetry.instrumentation.test.RetryOnAddressAlreadyInUseTrait
 import io.opentelemetry.instrumentation.test.utils.PortUtils
 import io.opentelemetry.semconv.trace.attributes.SemanticAttributes
-import io.opentelemetry.testing.armeria.client.WebClient
+import io.opentelemetry.testing.internal.armeria.client.WebClient
```

Comment:
what's the purpose of moving these to `internal` package if they still have external usages? or is this a first step?

Reply:
By external usage do you mean the usages in this repo? `internal` is fine to use in this repo, similar to the `internal` package being used across artifacts in the `opentelemetry-java` repo. The purpose is to make them unsupported for use from consumers of the `testing-common` package